packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -183,6 +194,22 @@ const ConnectorDetails = () => {
           />
         </Card>
       )}
+      {data && isSocial && inUse && (
```

Comment:
I don't think we need this check since the logic is independent

Reply:
updated.